cookbooks/cdo-apps/recipes/crontab.rb

```diff
@@ -1,17 +1,61 @@
 # Manages the user crontab for scheduled apps-related tasks.
+root = File.join node[:home], node.chef_environment
+bin = File.join root, 'bin'
 
-template "/home/#{node[:current_user]}/#{node.chef_environment}/crontab" do
-  source 'crontab.erb'
-  user node[:current_user]
-  group node[:current_user]
-  mode '0644'
-  notifies :run, 'execute[install-crontab]', :immediately
+def cronjob(m:nil, at:nil, cmd:, h:nil, notify:'')
```

Comment:
Would it make sense to put the m argument following the h argument?   (I realize these are named argument so they can be supplied in any order, but the seemingly arbitrary ordering here made it non-obvious what "m" and "h" even stand for.)